docs/articles/toc.yml

```diff
@@ -6,18 +6,22 @@
     href: intro/what-problems-does-actor-model-solve.md
   - name: Akka.NET Libraries and Modules
     href: intro/modules.md
-  - name: Part 1. Top-level Architecture
-    href: intro/tutorial-1.md
-  - name: Part 2. The Device Actor
-    href: intro/tutorial-2.md
-  - name: Part 3. Device Groups and Manager
-    href: intro/tutorial-3.md
-  - name: Part 4. Querying a Group of Devices
-    href: intro/tutorial-4.md
   - name: Use-case and Deployment Scenarios
     href: intro/use-case-and-deployment-scenarios.md
   - name: Production Users and Use Cases for Akka.NET
     href: intro/akka-users.md
+  - name: Getting Started
```

Comment:
To put all tutorials under this section

Reply:
Ah, so rather than have one giant `toc.yml` here we should probably have each folder use its own `toc.yml` and then just reference that from the main `articles/toc.yml` - we need to add this as part of our [DocFx Hygiene Guidelines](https://getakka.net/community/contributing/documentation-guidelines.html#docfx-hygiene)

Reply:
Got it